functests/utils/client/clients.go

```diff
@@ -3,28 +3,62 @@ package client
 import (
 	"os"
 
-	"github.com/golang/glog"
+	perfApi "github.com/openshift-kni/performance-addon-operators/pkg/apis"
+	configv1 "github.com/openshift/api/config/v1"
 	clientconfigv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+	mcov1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 	clientmachineconfigv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
+	sriovk8sv1 "github.com/openshift/sriov-network-operator/pkg/apis/k8s/v1"
+	sriovv1 "github.com/openshift/sriov-network-operator/pkg/apis/sriovnetwork/v1"
 	ptpv1 "github.com/openshift/ptp-operator/pkg/client/clientset/versioned/typed/ptp/v1"
 
+	"github.com/golang/glog"
 	discovery "k8s.io/client-go/discovery"
+	"k8s.io/client-go/kubernetes/scheme"
 	appsv1client "k8s.io/client-go/kubernetes/typed/apps/v1"
 	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	networkv1client "k8s.io/client-go/kubernetes/typed/networking/v1"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // Client defines the client set that will be used for testing
 var Client *ClientSet
 
 func init() {
+	// Setup Scheme for all resources
+	if err := perfApi.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
Instead of adding to scheme.Scheme, you can have a local scheme from it where you add yours and pass to the client:

```
        myScheme := runtime.NewScheme()
	scheme.AddToScheme(myScheme)
        configv1.AddToScheme(myScheme)
       ....
        clientSet.Client, err = client.New(config, client.Options{
		Scheme: scheme.Scheme,
	})
```

In this way the scheme you use is local and you don't tamper the global one (which is used also by the reporter).

This can be moved from init to `New`

Reply:
done